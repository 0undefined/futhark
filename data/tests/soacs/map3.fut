// --
// input {
//   [1,2,3]
// }
// output {
//   [2, 3, 4]
// }
fun [{int}] main([int] a) = map(fn {int} (int x) => {x+1}, a)
