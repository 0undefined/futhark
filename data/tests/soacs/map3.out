{Checked, [2, 3, 4]}
